docs/api-reference/geo-layers/h3-hexagon-layer.md

```diff
@@ -103,6 +103,38 @@ Possible values:
 * `false`: The layer chooses the mode automatically. High-precision rendering is only used if an edge case is encountered in the data.
 * `true`: always use high-precision rendering.
 
+##### `lowPrecision` (Boolean, optional)
+
+* Default: `false`
+
+Used to draw a large number of hexagons efficiently. Forces `H3HexagonLayer` to use low-precision instanced drawing by assuming that all hexagons within the current viewport have the same shape as the one at the center of the current viewport. The discrepancy is usually too small to be visible for resolutions higher than 5.
+
+Possible values:
+
+* `false`: The layer chooses the rendering mode automatically.
+* `true`: Forces low-precision drawing regardless of resolution of the data and other factors. Has a priority over `highPrecision` property.
+
+##### `resolution` (number, optional)
+
+* Default: `-1`
+
+Resolution of the data. It's possible to save some initialization time by passing the resolution, especially for large datasets. Values other than `-1` indicate that the data has only one resolution.
+
+Possible values:
+* `-1`: Indicates that the resolution is unknown and should be calculated.
+* `[0, 15]`: Valid h3 resolution.
+
+##### `hasPentagon` (boolean | null, optional)
```

Comment:
I don't think you need these two props if you:

- make the layer defer calculating these flags until it needs to
- always force high/low res rendering from the user application

Reply:
I think we'd still need `resolution`, which is currently inferred from data and used to get the cell used for instanced rendering. Otherwise you would need to infer that `lowPrecision: true` implied a single resolution, and loop through the data for the first valid cell to determine what that resolution was.

Reply:
Removed resolution and hasPentagon props. 
For highPrecision: false - select first resolution and exit the loop early.